core/pta/imx/dek_blob.c

```diff
@@ -60,6 +61,13 @@ static TEE_Result do_generate(const uint8_t *payload, size_t payload_size,
 	assert(payload_size == (128 / 8) || payload_size == (192 / 8) ||
 	       payload_size == (256 / 8));
 
+	/*
+	 * Prevent against an unexpected padding of dek_blob_header structure
+	 * that must remain packed. This structure will be seriailized to a
+	 * buffer along the DEK blob.
+	 */
+	assert(sizeof(struct dek_blob_header) == 8 * sizeof(uint8_t));
```

Comment:
This should be a `static_assert()` to catch the error at build time.

Reply:
Ok